router/v1/public.go

```diff
@@ -88,7 +88,7 @@ func (h *Handlers) GetPublicEmojiJSON(c echo.Context) error {
 }
 
 func generateEmojiJSON(repo repository.StampRepository, buf *bytes.Buffer) error {
-	stamps, err := repo.GetAllStamps(false)
+	stamps, err := repo.GetAllStamps(repository.StampTypeOriginal)
```

Comment:
```suggestion
	stamps, err := repo.GetAllStamps(repository.StampTypeAll)
```
`excludeUnicode=false`に対応するのは`repository.StampTypeAll`っぽそうです